contracts/interchain-token-service/src/shared.rs

```diff
@@ -25,6 +26,22 @@ impl TryFrom<u32> for NumBits {
     }
 }
 
+impl NumBits {
+    // rounds up to the nearest supported number of bits.
+    // Anything greater than 256 is rounded to 256
+    pub fn round_to_nearest<T>(value: T) -> Self
+    where
+        T: Into<Uint256>,
+    {
+        match value.into() {
```

Comment:
Why are we only allowing these specific values for max_uint_bits? E.g. stellar uses uint127